configuration_helper/models/config.py

```diff
@@ -0,0 +1,54 @@
+# -*- coding: utf-8 -*-
+# © 2014 David BEAL Akretion
+# © 2016 Yannick Vaucher (Camptocamp SA)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+import re
+
+from openerp import api, fields, models
+
+
+class AbstractConfigSettings(models.AbstractModel):
+    _name = 'abstract.config.settings'
+    _description = 'Abstract configuration settings'
+    # prefix field name to differentiate fields in company with those in config
+    _prefix = 'setting_'
+    # this is the class name to import in your module
+    # (it should be ResCompany or res_company, depends of your code)
+    _companyObject = None
```

Comment:
You can break compatibility here to become cleaner, no problem between versions. 
There was no version for V8, only v7.